_posts/2024-10-02-#30793.md

```diff
@@ -10,13 +10,53 @@ status: upcoming
 commit:
 ---
 
-_Notes and Questions to follow soon!_
+## Notes
 
-<!-- ## Notes -->
+- An orphan transaction is a transaction with missing inputs. The p2p code uses a [`TxOrphanage`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L1079-L1081) to store orphan transactions, to be reconsidered later if/when its parent transaction(s) are known.
 
-<!-- ## Questions -->
+    - We have discussed `TxOrphanage` in previous review club meetings [n21527](/21527) and [n30000](/30000).
 
-<!-- 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach? -->
+- [PR #30793](https://github.com/bitcoin/bitcoin/pull/30793) adds a new RPC, `getorphantxs`, to
+  return the contents of the node's orphanage at that moment.
+
+    - Its format is similar to the `getrawmempool` RPC, which also returns information on all transactions in the mempool. Lower verbosity returns the txids,
+      and higher verbosity returns fields about each entry.
+
+    - Its purpose is similar to that of `getrawaddrman`. Most likely, developers will be the main users.
+
+## Questions
+
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
+
+1. What is an orphan transaction? At what point do transactions enter the orphanage (can you find the code)?
+
+1. What command can you run to get the list of available RPCs?
```

Comment:
Maybe too obvious, but potentially helpful to help prime for the later `hidden` question.

```diff
- 1. What command can you run to get the list of available RPCs?
+ 1. What command can you run to get the list of available RPCs? Why don't you see `getorphantxs` in the list?
```

Reply:
Gonna keep this one quiet 🤭 maybe can be a hint if people don't get the later question